contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,133 @@
+- For a site whose main goal is to attract SEO traffic and is looking to improve their [PageSpeed Insights performance score](https://pagespeed.web.dev/), then Page Loaded will be the most important since this contributes the most to [Total Blocking Time](https://web.dev/tbt/), which has the [highest weighting](https://developer.chrome.com/docs/lighthouse/performance/performance-scoring/?utm_source=lighthouse&utm_medium=lr#weightings) when calculating the performance score.
+- For an e-commerce site with a checkout form with many input fields and complex validation, a high DOM Interactive time means that the form is slow to respond to user input. 
+- For a personal finance app, DOM Interactive ensures that users can quickly navigate and interact with various financial tools on offer. However, Page Loaded guarantees that all the data visualizations, such as charts and graphs, are fully loaded and functional. 
+
+## Optimization cheat sheet
+If you're looking to improve any of the above metrics, here is a handy cheat sheet for where to look for opportunities. Each value in the table indicates how likely the metrics are to be affected.
+
+| What?   | First Contentful Paint  | DOM Interactive | Page Loaded |
+|--------|--------|--------|--------|
+| Page Size   | 🟢 High  | 🟢 High   | 🟢 High |
+| Server Response Time   | 🟢 High  | 🟢 High | 🟢 High |
+| Network Latency   | 🟢 High  | 🟢 High | 🟢 High |
+| Downloading JavaScript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
+| Executing JavaScript  | 🟡 Medium (only if it blocks rendering) | 🟢 High | 🟢 High |
+|API Requests |	🟡 Medium (only if they block rendering or are critical for initial content) |	🟡 Medium (only if they are for rendering critical content) |	🟢 High |
+| CSS Files: Downloading and Parsing  | 🟢 High | 🟡 Medium | 🟡 Medium |
+| Images: Downloading and Rendering| 🟡 Medium (only if they are part of the initial content) | 🔴 Low | 🟢 High |
+| Fonts: Downloading and Rendering  | 🟡 Medium (only if they are part of the initial content)  | 🔴 Low |  🔴 Low  |
+
+## Identifying opportunities on your own site
+PostHog's performance monitoring tool shows you the size and speed of different requests made during page load. You can also filter these by request type, such as API, JS, image, and CSS requests.
+
+Combine this with the cheat sheet above and you'll be able to pinpoint what are the contributing factors to your metric times.
+
+### **Examples**
+**1. API requests**
+![A screenshot of the network tab filtered to show API requests only. One API request is identified as taking too long.](https://i.imgur.com/JPeZeTe.png)
+
+Here we can see an API request that is happening during page load (indicated by the `LOAD` text next to it). Since it is blocking the initial rendering, it is contributing to the high FCP time as well as the Page Loaded time.
```

Comment:
This isn't actually true... It's a weird edge case of the way our player works. LOAD simply means that the request (or event or whatever) took place _before_ the recording started. Probably the wrong term given how it conflates with the Page load etc. 

_Also_ it might not be contributing to the high FCP or Load, as it is an XHR request which generally means it is happening _after_ the page is loaded and running. The little blue line indicates when the request started and finished which is after the page load event. This is definitely not-obvious so I think we have some good follow up changes to make both to the `LOAD` term and perhaps a way to indicate which requests are and aren't related to the top metrics...